src/main/java/com/group8/library_management/controller/BookController.java

```diff
@@ -0,0 +1,30 @@
+package com.group8.library_management.controller;
+
+import com.group8.library_management.dto.response.PageRes;
+import com.group8.library_management.dto.response.*;
+import com.group8.library_management.dto.request.BookSearchReq;
+import com.group8.library_management.service.BookService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/books")
+@RequiredArgsConstructor
+public class BookController {
+
+    private final BookService bookService;
+
+    @PostMapping("/search")
+    public ResponseEntity<?> searchBooks(
+            @RequestBody BookSearchReq request,
+            @RequestParam(defaultValue = "0") int page,
+            @RequestParam(defaultValue = "10") int size
+    ) {
+        PageRes<BookDetailRes> result = bookService.searchBooks(request, page, size);
+
+        return ResponseEntity.ok(
+                BaseAPIRes.success(200,"Book list:", result)
```

Comment:
```suggestion
                BaseAPIRes.success(200, "Book list:", result)
```
200 
chị thcihs đó, cơ mà có http code kìa, ai chơi set cứng ri 
=> tạo class constant mà dùng  gom 1 nơi dễ chỉnh sửa

Reply:
Dạ em đã tạo folder /constant rồi ạ